### PR TITLE
Created Invite.yml

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
Automates invitation process
Addresses #162

### CODE REVIEW

File: .github/workflows/invite.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  automate_invite:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Invite on label
+        uses: vj-abigo/invite-on-label@v1.2
+        with:
+          organization: SidHacksCommunity
+          label: invite me to the organisation
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          comment: '<b>Invitation sent for the GitHub Organisation. Welcome to the CodXCrypt community 🎉</b>'
```
Comment (Panquesito7) [inline .github/workflows/invite.yml:15]:
```suggestion
          comment: '<b>Invitation sent for the GitHub Organisation. Welcome to the SidHacks community! 🎉</b>'
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Comment by Panquesito7** [comment]
Ping. @siddharthdayalwal.

**Comment by siddharthdayalwal** [comment]
Hey @Panquesito7 

Thanks for the ping! 

Could you automate it with the reply as you can see here: https://github.com/SidHacksCommunity/support/issues/2#issuecomment-891585321 ?

**Comment by Panquesito7** [comment]
> Hey @Panquesito7
> 
> Thanks for the ping!
> 
> Could you automate it with the reply as you can see here: [#2 (comment)](https://github.com/SidHacksCommunity/support/issues/2#issuecomment-891585321) ?

Should I open another PR for it or tell @AbhijithGanesh to update it?

**Comment by siddharthdayalwal** [comment]
Let's wait for @AbhijithGanesh to jump in if he could do it before you do it. :)

**Comment by AbhijithGanesh** [comment]
Can you tell me what I should do

**Comment by Panquesito7** [comment]
> Can you tell me what I should do

https://github.com/SidHacksCommunity/support/issues/2#issuecomment-891585321

**Pull request closed without merging**